DigitalLearningSolutions.Data/DataServices/UserDataService/DelegateUserCardDataService.cs

```diff
@@ -41,8 +41,7 @@ public partial class UserDataService
                         AND au.CentreID = cd.CentreID
                         AND au.Email != ''
                         AND au.Active = 1
-                ) AS AdminID,
-                cd.AliasID

```

Comment:
Just wanted to point out that you've missed a reference to cd.AliasID a few lines above in this query, but that's fine because I eliminated that in 897